frontend/src/components/school/SchoolDetails.vue

```diff
@@ -0,0 +1,501 @@
+<template>
+  <v-container class="containerSetup" fluid>
+    <div style="width: 100%;" :overlay=false>
+      <v-row class="pt-0"
+             :class="{'mr-0 ml-0': $vuetify.breakpoint.smAndDown, 'mr-3 ml-3': $vuetify.breakpoint.mdAndUp}">
+        <v-col cols="10 pt-0">
+          <div>
+          <!--  <v-row> -->
+              <!--<v-col class="pb-0 pt-0 d-flex justify-start"> -->
```

Comment:
Commented code. 

Cols should be within a row ex. <row> <col> 

if you face padding issues try no-gutters in row. ex. <row no-gutters>